src/index.ts

```diff
@@ -34,6 +35,7 @@ export class Probot {
   private app?: OctokitApp
   private githubToken?: string
   private throttleOptions: any
+  private octokitConstructor?: Octokit.Static
```

Comment:
I would call the option `Octokit` with a capital O.

And I would always default it to @octokit/rest, so that we don’t need to do further checks

Reply:
Should the default be your `ProbotOctokit`?

Reply:
Yes